docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,9 @@
+# Broadcast rules for elementwise ops {#openvino_docs_ops_broadcast_rules}
```

Comment:
broadcast_rules.md is introduced with [4366](https://github.com/openvinotoolkit/openvino/pull/4366/), it is not required to review it in 2 different commits.

Reply:
It's here to have good doc generations. I'll live it empty in this PR.